app/templates/components/tumblr-post.hbs

```diff
@@ -1,6 +1,15 @@
-{{tumblr-post-text
-  post=post
-  postsRoute=postsRoute
-  collapsible=collapsible
-  collapsed=collapsed
-  formatDates=formatDates}}
\ No newline at end of file
+{{#if post.isPhotoEntry}}
```

Comment:
Let's use [`ember-truth-helpers`](https://github.com/jmurphyau/ember-truth-helpers) here:

`{{#if (eq post.type 'photo')}}`

You can install this addon to the repo:

`ember install ember-truth-helpers`

Please make sure to move it to `dependencies` in package.json; by default, it will be added to `devDependencies`.

Reply:
I was aware of ember-truth-helpers but didn't want to add new dependencies to the project.  Will add it thanks.